### ISSUE TITLE
SystemJS: Live bindings break with +=

### ISSUE THREAD

**Issue by rixo** [rollup/rollup#3238]
- Rollup Version: 1.27.0
- Operating System (or Browser): n.a.
- Node Version: n.a.

### How Do We Reproduce?

[REPL] (systemjs tab)

[REPL]: https://rollupjs.org/repl/?version=1.27.0&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmV4cG9ydCUyMGxldCUyMGZvbyUyMCUzRCUyMDElNUNuJTVDbiUyRiUyRiUyMG9rJTVDbmZvbyUyMCUzRCUyMGZvbyUyMCUyQiUyMDElNUNuJTVDbiUyRiUyRiUyMG5vdCUyMG9rJTVDbmZvbyUyMCUyQiUzRCUyMDElMjIlMkMlMjJpc0VudHJ5JTIyJTNBdHJ1ZSU3RCU1RCUyQyUyMm9wdGlvbnMlMjIlM0ElN0IlMjJmb3JtYXQlMjIlM0ElMjJzeXN0ZW0lMjIlMkMlMjJuYW1lJTIyJTNBJTIybXlCdW5kbGUlMjIlMkMlMjJhbWQlMjIlM0ElN0IlMjJpZCUyMiUzQSUyMiUyMiU3RCUyQyUyMmdsb2JhbHMlMjIlM0ElN0IlN0QlN0QlMkMlMjJleGFtcGxlJTIyJTNBbnVsbCU3RA==

### Expected Behavior

~~~js
foo += 1
~~~

should transpile to:

~~~js
foo = exports('foo', foo + 1);
~~~

### Actual Behavior

it transpiles to:

~~~js
foo += exports('foo', 1);
~~~

**Comment by lukastaegert** [comment]
Thanks for spotting!

**Comment by lukastaegert** [comment]
Fix at #3242
